apps/website/src/pages/insights/repos.tsx

```diff
@@ -53,12 +53,12 @@ const ReposPage: Page = () => {
         </Text>
       </div>
 
-      <div className="grid grid-cols-4 gap-6">
+      <div className="grid grid-cols-3 gap-5">
         {repos.map(repo => (
           <Link
             key={repo.name}
             href={`https://github.com/status-im/${repo.name}`}
-            className="border-neutral-10 hover:border-neutral-40 flex flex-col rounded-2xl border px-4 py-3 transition-colors duration-200"
+            className="border-neutral-10 hover:border-neutral-40 flex h-[124px] flex-col rounded-2xl border px-4 py-3 transition-colors duration-200"
             target="_blank"
             rel="noopener noreferrer"
```

Comment:
I don't think you need this since you're already checking if it is an external link inside the Link component.

```
{...(external && {
        target: '_blank',
        rel: 'noopener noreferrer',
      })}
```

Reply:
resolved in https://github.com/status-im/status-web/pull/392/commits/5aeca5aa443a1f0cff83f40b91d051a7f0b18054